AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -33,7 +34,9 @@ struct SessionHeaderView: View {
         }
         .font(Font.moderate(size: 13, weight: .regular))
         .foregroundColor(.aircastingGray)
-        
+        .onAppear() {
+            print("\(session.name) status is: \(session.status)")
```

Comment:
Please use `Log.info` and make the log more verbose. Give more context to it. You know what is this log but You'll probably forget what was it doing very fast. Be kind to other developers and to Your future self and let us know what is happening. if the session.name would be `foo` the log would be `foo status is recoding`. maybe `microfon session "foo" status is recording` or similar depending on what exactly is happening.